llvm/lib/Target/X86/X86ISelLowering.cpp

```diff
@@ -654,6 +655,7 @@ X86TargetLowering::X86TargetLowering(const X86TargetMachine &TM,
       setOperationAction(ISD::FSIN   , VT, Expand);
       setOperationAction(ISD::FCOS   , VT, Expand);
       setOperationAction(ISD::FSINCOS, VT, Expand);
+      setOperationAction(ISD::FTAN, VT, Expand);
```

Comment:
I want to tell you to match the existing code that uses extra spaces to keep things in columns, but I guess clang-format will complain.

Reply:
Exactly, the format is changed because of clang-format. Without changing the github action, I don't see how I can preserve the columns spacing.